app/src/main/java/org/mozilla/fenix/settings/PairFragment.kt

```diff
@@ -35,13 +39,17 @@ class PairFragment : Fragment(R.layout.fragment_pair), BackHandler {
                 onNeedToRequestPermissions = { permissions ->
                     requestPermissions(permissions, REQUEST_CODE_CAMERA_PERMISSIONS)
                 },
-                onScanResult = { pairingUrl ->
-                    requireComponents.services.accountsAuthFeature.beginPairingAuthentication(
-                        requireContext(),
-                        pairingUrl
-                    )
-                    findNavController(this@PairFragment)
-                        .popBackStack(R.id.turnOnSyncFragment, false)
+                onScanResult = {
+                        pairingUrl ->requireComponents.services.accountsAuthFeature.beginPairingAuthentication(requireContext(),pairingUrl)
+                    val vibrator = context?.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                        vibrator.vibrate(VibrationEffect.createOneShot(200, VibrationEffect.DEFAULT_AMPLITUDE))
+                    } else {
+                        //vibrator.vibrate(200)  //commented to avoid unknown gradle error
```

Comment:
Can you please fix this. We cannot let any gradle errors affect the vibration.

Reply:
Adding `@Suppress("Deprecation")` should probably fix the problem.